examples/getting_started/example_config.yaml

```diff
@@ -1,146 +1,247 @@
-modalities_setup:
-  run_mode: FROM_SCRATCH
-  settings: 
-    global_num_seen_samples: 0
+settings:  
+  experiment_id: ${modalities_env:experiment_id}
+  referencing_keys:
+    sample_key: input_ids
+    target_key: target_ids
+  training:
+    global_training_log_interval_in_steps: 48
+    global_checkpointing_interval_in_steps: 48
+    global_evaluation_interval_in_steps: 48
+    global_num_seen_steps: 0
+    do_apply_activation_checkpointing: true
+    gradient_acc_steps: 1
+    local_train_micro_batch_size: 8
+    sequence_length: 1024
```

Comment:
maybe 512?

Reply:
Fixed in e391cfc2646dcb52e9943449cc42f665f3b3b5fc.